src/channel-picker/channel-picker.js

```diff
@@ -25,14 +25,14 @@ const Self = class ChannelPicker extends ColorElement {
 	connectedCallback () {
 		super.connectedCallback?.();
 
-		this._el.picker.addEventListener("change", this);
+		this._el.picker.addEventListener("change", this, {capture: true});
```

Comment:
Why are we using `change` here and not `input`?

Reply:
IDK why I did it. Probably, there was some misunderstanding. I checked the description of both events on MDN and see no reason to listen to `change` instead of `input` (we want to monitor every change of the picker value, not only the committed ones). I'll fix it. Thanks!